additional-skills/git/README.md

```diff
@@ -0,0 +1,78 @@
+# Git + GitHub
+
+All sections are required.
+
+## Initializing 
+
+* Demonstrate initializing a new git repository
+* Demonstrate creating an empty root commit
+* Can explain the reasoning for creating an empty root commit
+
+## Basics
+
+* Demonstrate usage of `git status`
+* Demonstrate staging files for commit
+* Demonstrate committing staged files
+* Demonstrate committing a subset of changed files using staged and unstaged files
+* Demonstrate using stash and stash pop
+
+## Commits, refs, and labels 
+
+* Can explain what a commit ref is
+* Can explain what a tag is
+* Can explain what a branch is
+
+## Diffs
+
+* Demonstrate viewing the diff of changes
+* Demonstrate viewing staged changes
+
+## Branches
+
+* Demonstrate creating a new branch
+* Demonstrate deleting a new branch
+* Demonstrates good branch naming conventions (i.e. lowercase using dashes and/or slashes)
+* Demonstrate changing branches
+* Demonstrate using stash to change branches when there are conflicts with uncommitted changes
+
+## Reset
+
+* Demonstrate using a hard reset with a ref
+* Demonstrate using a soft reset with a ref 
+* Explain why you'd use a hard reset or a soft reset
+
+## Merging
+
+* Demonstrate merging two refs to create a new ref
+* Demonstrate resolving merge conflicts
+
+## Rebasing
+
+* Demonstrate doing a simple rebase of master onto a branch 
+* Demonstrate performing an amend to previous commit
+* Demonstrate performing an interactive rebase of a branch
+  * Demonstrate using all possible rebase options
+
+## Visualizing Commits
+
+* Can draw a git tree on paper
+* Can explain a rebase with appropriate drawing
+* Can explain a merge with appropriate drawing
+
+## Working with Remotes
+
+* Demonstrate adding a remote to an existing repository
+* Demonstrate pushing to a remote repository
+* Explain the difference between `git pull` and `git fetch`
+* Explain the difference between `origin/master` and `master`
+* Demonstrate setting the default upstream branch for a local branch
+* Demonstrate force pushing rebases into isolated upstream Pull Request branches
+
+## GitHub
```

Comment:
I couldn't think of anything else of interest on here. Maybe commenting?

Reply:
`Demonstrate appropriate rebasing of isolated branches` assuming this means rebasing/squashing commit history?

Reply:
yea

Reply:
Don't merge a branch with 50 commits

Reply:
Words to live by.

Reply:
Perhaps something around PR etiquette, but I don't really have anything to add. LGTM!